src/Symfony/Component/HttpKernel/Attribute/AsController.php

```diff
@@ -12,7 +12,11 @@
 namespace Symfony\Component\HttpKernel\Attribute;
 
 /**
- * Service tag to autoconfigure controllers.
+ * Autoconfigures controllers as services by applying the `controller.service_arguments`
+ * tag to them. This allows to inject services in controller action methods, not
+ * only in their constructors.
+ * 
+ * @see https://symfony.com/doc/current/controller/service.html
```

Comment:
WDYT of this? (note that we don't link to the doc usually).

```suggestion
 * tag to them. This enables injecting services as method arguments
 *  in addition to other conventional dependency injection strategies.
```

Reply:
It is better I think, thank you for your feedback !